### PR TITLE
feat: add elastic search handlers

### PR DESCRIPTION
**Pull request by Ali-Salman29**
- Add elastic search handlers on create, update, and delete methods
- Updated temporary indexes to fetch recent aliases from the elastic search
- Fix Thread Serializer issues

### CODE REVIEW

File: forum/handlers.py [hunk 1]
```diff
@@ -0,0 +1,110 @@
+"""
+Handlers for the forum app.
+"""
+
+import logging
+from typing import Any, Dict
+
+from forum.utils import get_str_value_from_collection
+from forum.search.es_helper import ElasticsearchHelper
```
Comment (Faraz32123) [inline forum/handlers.py:9]: import order

Comment (Ali-Salman29) [reply forum/handlers.py:9]: Could you please elaborate?

Comment (Faraz32123) [reply forum/handlers.py:9]:
```
+from forum.search.es_helper import ElasticsearchHelper
 from forum.utils import get_str_value_from_collection
-from forum.search.es_helper import ElasticsearchHelper
```

**Review by taimoor-ahmed-1 — CHANGES_REQUESTED** [review]

File: forum/utils.py [hunk 2]
```diff
@@ -52,8 +53,50 @@ def get_int_value_from_collection(
...
+    elif name == "comment_inserted":
+        return signals.comment_inserted
+    elif name == "comment_thread_inserted":
+        return signals.comment_thread_inserted
+    elif name == "comment_updated":
+        return signals.comment_updated
+    elif name == "comment_thread_updated":
+        return signals.comment_thread_updated
+    else:
+        raise ValueError(f"No signal found for name: {name}")
```
Comment (Faraz32123) [inline forum/utils.py:102]: we can make a dict and return a matched value
```
signals_map = {
        "comment_deleted": signals.comment_deleted,
        "comment_thread_deleted": signals.comment_thread_deleted,
        "comment_inserted": signals.comment_inserted,
        "comment_thread_inserted": signals.comment_thread_inserted,
        "comment_updated": signals.comment_updated,
        "comment_thread_updated": signals.comment_thread_updated,
    }

    try:
        return signals_map[name]
    except KeyError:
        raise ValueError(f"No signal found for name: {name}")
```

**Review by Faraz32123 — APPROVED** [review]
LGTM!

**Review by taimoor-ahmed-1 — APPROVED** [review]

**Pull request merged**